src/camera.py

```diff
@@ -0,0 +1,212 @@
+"""
+camera
```

Comment:
Since you're adding this module documentation, you should also go enable it in the Sphinx files. You can't generate the docs without the car, but it's really simple to add it. Take a look at the other modules (rst files I believe).